streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/kstream/KStream.scala

```diff
@@ -211,7 +213,7 @@ class KStream[K, V](val inner: KStreamJ[K, V]) {
    * }}}
    *
    * @param topic the topic name
-   * @param (implicit) produced the instance of Produced that gives the serdes and `StreamPartitioner`
```

Comment:
Why removing `implicit`?

Reply:
It's a scaladoc syntax error.

Reply:
We could add it after `produced`, but AFAICT, the param name must immediately follow `@param`

Reply:
unless my IDE is also wrong about this...

Reply:
You can build the scala docs and double check if in doubt -- if is works same a JavaDocs, I think John is correct.

Reply:
Confirmed: when I generate the scaladoc from trunk, it just skips this parameter and only documents `topic`, but when I generate it from this branch, it documents both `topic` and `produced`.